### PR TITLE
More playpen style corrections (deis-workflow-cli, heroku, imagemagick-stable, imagemagick-edge)

### PR DESCRIPTION
**Pull request by tsimonq2**
Fixed the following snaps:
- deis-workflow-cli
- heroku
- imagemagick-stable
- imagemagick-edge

### CODE REVIEW

File: deis-workflow-cli/snapcraft.yaml [hunk 1]
```diff
@@ -13,6 +13,6 @@ apps:
 
 parts:
   workflow-cli:
-    plugin: go
     source: https://github.com/deis/workflow-cli
     source-type: git
```
Comment (didrocks) [inline deis-workflow-cli/snapcraft.yaml]: this is optional, add .git to source ;)

File: imagemagick-edge/snapcraft.yaml [hunk 2]
```diff
@@ -14,49 +14,48 @@ confinement: devmode
...
+        plugs: [home, unity7, x11]
     stream:
         command: stream
-        plugs: [ home, unity7, x11 ]
+        plugs: [home, unity7, x11]
 
 parts:
     imagemagick:
-        plugin: autotools
         source: git://github.com/ImageMagick/ImageMagick
```
Comment (didrocks) [inline imagemagick-edge/snapcraft.yaml]: missing .git to source:

File: imagemagick-stable/snapcraft.yaml [hunk 3]
```diff
@@ -14,43 +14,42 @@ confinement: devmode
...
+        plugs: [home, unity7, x11]
     stream:
         command: stream
-        plugs: [ home, unity7, x11 ]
+        plugs: [home, unity7, x11]
 
 parts:
     imagemagick:
-        plugin: autotools
         source: http://git.imagemagick.org/repos/ImageMagick/repository/archive.tar.bz2?ref=ImageMagick-6
```
Comment (didrocks) [inline imagemagick-stable/snapcraft.yaml]: did you try that one? I'm unsure snapcraft will be able to autodetect that one

File: imagemagick-edge/snapcraft.yaml (hunk 2 above)
Comment (tsimonq2) [inline imagemagick-edge/snapcraft.yaml]: adding

File: imagemagick-stable/snapcraft.yaml (hunk 3 above)
Comment (tsimonq2) [inline imagemagick-stable/snapcraft.yaml]: I'll do a test build quick

Comment (tsimonq2) [inline imagemagick-stable/snapcraft.yaml]: @didrocks works fine

**Comment by didrocks** [comment]
👍

**Pull request merged**